frontend/public/components/marketplace/kubernetes-marketplace-items.jsx

```diff
@@ -0,0 +1,323 @@
+  renderTiles() {
+    const items = this.state.filteredItems;
+
+    return (
+      <CatalogTileView.Category totalItems={items.length} viewAll={true}>
+        {_.map(items, ((item, index) => {
+          const { tileName, tileImgUrl, tileIconClass, tileProvider, tileDescription } = item;
+          const uid = `${tileName}-${index}`;
+          const iconClass = tileIconClass ? `icon ${normalizeIconClass(tileIconClass)}` : null;
+          const vendor = tileProvider ? `Provided by ${tileProvider}` : null;
+          return <CatalogTile
+            id={uid}
+            key={uid}
+            title={tileName}
+            iconImg={tileImgUrl}
+            iconClass={iconClass}
+            vendor={vendor}
+            description={tileDescription}
+          />;
+        }))}
+      </CatalogTileView.Category>
+    );
+  }
+
+  render() {
+    const { filters, filterCounts } = this.state;
+    const items = this.state.filteredItems;
+
+    return (
+      <div className="co-catalog-page">
+        <div className="co-catalog-page__tabs">
+          <FilterSidePanel>
+            {_.map(filters, ((filter, category) => {
+              if (category === 'byName') {
+                return (<FilterSidePanel.Category key={category}>
+                  <FormControl type="text" placeholder="Filter by name..." bsClass="form-control"
+                    value={filters.byName.value}
+                    onChange={e => this.onFilterChange('byName', null, e.target.value)}
+                  />
+                </FilterSidePanel.Category>);
+              }
+              let title = category;
+              if (title.startsWith('by')) {
+                title = title.replace(/by/, '');
+              }
+              if (title.startsWith('tile')) {
+                title = title.replace(/tile/, '');
+              }
+              return (<FilterSidePanel.Category
+                key={category}
+                title={title}
+              >
+                {_.map(filter, ((filterItem, categoryItem) => {
+                  const label = filterItem.label;
+                  return (<FilterSidePanel.CategoryItem
+                    key={categoryItem}
+                    count={filterCounts[category][categoryItem]}
+                    checked={filterItem.active}
+                    onChange={e => this.onFilterChange(category, categoryItem, e.target.checked)}
+                  >
+                    {label}
+                  </FilterSidePanel.CategoryItem>);
+                }))}
+              </FilterSidePanel.Category>);
+            }))}
+          </FilterSidePanel>
+        </div>
+        <div className="co-catalog-page__content">
+          <div>
+            <div className="co-catalog-page__num-items">{_.size(items)} items</div>
+          </div>
+          {items.length > 0 && <CatalogTileView>
+            {this.renderTiles()}
+          </CatalogTileView>}
+          {items.length === 0 && <EmptyState className="blank-slate-content-pf">
```

Comment:
Adding `blank-slate-content-pf` is the right thing to do, but won't have effect until we upgrade `patternfly` to at least `3.54.2`. This is why the background on the empty state is still grey.

Reply:
To be consistent with other EmptyState pages in console, please see comments by @tlwu2013 starting [HERE](https://github.com/openshift/console/pull/642#issuecomment-429077646).
Basically,
- transparent background
- no border or padding
- Title and subtext modifications